doc/01_intro.md

```diff
@@ -0,0 +1,37 @@
+# Introduction {#mainpage} #
+The purpose of a [synchronization client](@ref synchronization_client) for an AsteroidOS watch is to provide a means by which data and commands from the machine, generally, a computer or mobile phone running Linux on which the client runs, to the watch.  Although either a computer or mobile phone are commonly used for this, the rest of this document will refer to "the computer" to mean the computer on which the synchronization client runs.
```

Comment:
In my opinion, this sounds a bit vague, even though you try to clarify things. The use of _the machine_ makes it more strange?
At first read gives the impression that the _synchronization client_ runs on the watch, which isn't the case.

I would change this to start from an even simpler level.

Something like this maybe?
AsteroidOS synchronizes notifications, weather data, music metadata, etc to the watch using a synchronization client running on a Linux computer or a mobile phone.

Reply:
Really good point.  I tried a modification of your suggestion.  Please let me know what you think.